frontend/src/components/Teams/CreateTeam/CardMember/index.tsx

```diff
@@ -76,14 +77,26 @@ const CardMember = React.memo<CardBodyProps>(
             }}
           >
             <Flex align="center" css={{ width: '23%' }} gap="8">
-              <Icon
-                name="blob-personal"
-                css={{
-                  width: '32px',
-                  height: '$32',
-                  zIndex: 1,
-                }}
-              />
+              {isOpen ? (
+                <Icon
+                  name="blob-personal"
+                  css={{
+                    width: '32px',
+                    height: '$32',
+                    zIndex: 1,
+                    opacity: 0.2,
+                  }}
+                />
+              ) : (
+                <Icon
+                  name="blob-personal"
+                  css={{
+                    width: '32px',
+                    height: '$32',
+                    zIndex: 1,
+                  }}
+                />
+              )}
```

Comment:
```suggestion
                <Icon
                  name="blob-personal"
                  css={{
                    width: '32px',
                    height: '$32',
                    zIndex: 1,
                    opacity: isOpen ? 0.2 : 1,
                  }}
                />
```

I'm not sure if the opacity is 1 when the `isOpen` is false but I know you got the idea